include/nil/blueprint/blueprint/plonk/assignment.hpp

```diff
@@ -281,6 +309,12 @@ namespace nil {
                                     this->_private_table.witness(j)[i] : 0).data << " ";
                     }
                     os << "| ";
+                    for (std::uint32_t j = 0; j < shared_size; j++) {
```

Comment:
This would currently break the circuit visualizer, so I wouldn't really change this right now tbh.

Reply:
ok, done